torch/_dynamo/symbolic_convert.py

```diff
@@ -1015,9 +1036,15 @@ def CALL_FUNCTION_KW(self, inst):
         self.call_function(fn, args, kwargs)
 
     def LOAD_METHOD(self, inst):
-        self.LOAD_ATTR(inst)
-        self.push(self.pop())
-        self.push(None)
+        if sys.version_info >= (3, 11):
+            self.LOAD_ATTR(inst)
+            obj = self.pop()
+            self.PUSH_NULL(inst)
```

Comment:
This assumes that this never a proper method? Otherwise you shouldn't push a NULL here.